src/store/ordinoSlice.ts

```diff
@@ -83,6 +83,14 @@ export interface IWorkbench {
    */
   selection: IRow['_visyn_id'][];
 
+  formatting?: {
+    titleColumn?: string;
+    idColumn: string;
+    formatResult?: string;
```

Comment:
What does this mean? 🤔 Is it a template string? Please add a comment and add an example.